apps/api/src/schema/websoc.ts

```diff
@@ -118,30 +137,71 @@ export const websocQuerySchema = z.object({
       }
       return parsedNums;
     }),
-  instructorName: z.string().optional(),
-  days: daysSchema.optional(),
-  building: z.string().optional(),
-  room: z.string().optional(),
+  instructorName: z.string().optional().openapi({
+    description: "The instructor name of the course",
+    example: "Anteater, P.",
+  }),
+  days: daysSchema.optional().openapi({
+    description: "The days of the week for when the course meets",
+    example: "M,Tu,W,Th",
+  }),
+  building: z.string().optional().openapi({
+    description: "The building where the course is held",
+    example: "ALP",
+  }),
+  room: z.string().optional().openapi({
+    description: "The room where the course is held",
+    example: "1100",
+  }),
   division: z
     .enum(courseLevels)
     .or(z.literal("ANY"))
     .optional()
+    .openapi({
+      description:
+        "The type of course level of the course (e.g. Lower Division, Upper Division, or Graduate)",
+      example: "LowerDiv",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   sectionType: z
     .union([z.enum(anyArray), z.enum(websocSectionTypes)])
     .optional()
+    .openapi({
+      description: "The type of section of the course (e.g. Lecture, Discussion, Lab)",
+      example: "Lec",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   fullCourses: z
     .enum(fullCoursesOptions)
     .optional()
+    .openapi({
+      description: "The enrollment status of the course",
```

Comment:
This description is a bit unclear to me and since it's a query parameter, it shouldn't be phrased like it's a field of one course. Something like "How to filter courses by filled status" might be appropriate; workshop it and see what sounds right.